app/javascript/CommentSubscription/CommentSubscription.jsx

```diff
@@ -81,6 +83,7 @@ export class CommentSubscription extends Component {
       onSubscribe,
       onUnsubscribe,
       positionType = 'relative',
+      userStatus
```

Comment:
refactor (blocking) - we only use this to check if the user is logged in or not, so let's just pass in a `isLoggedIn` boolean instead to keep things simpler
```suggestion
      isLoggedIn
```

This prop also needs added to the PropTypes (starting line 240)